libraries/ArduinoOTA/ArduinoOTA.h

```diff
@@ -94,6 +94,7 @@ class ArduinoOTAClass
     void _onRx(void);
     int parseInt(void);
     String readStringUntil(char end);
+    String generateNonce(void);
```

Comment:
There is no need for this to be a member method. Please move it to be a static function in the .cpp, or put it in an anonymous namespace.